packages/modules/vehicles/ovms/soc.py

```diff
@@ -11,13 +11,12 @@
 from modules.vehicles.ovms import api
 from modules.vehicles.ovms.config import OVMS, OVMSConfiguration
 
-
+date_fmt = '%Y-%m-%d %H:%M:%S'
 log = logging.getLogger(__name__)
 
 
 def fetch(vehicle_update_data: VehicleUpdateData, config: OVMS, vehicle: int) -> CarState:
-    soc, range, soc_ts = api.fetch_soc(config, vehicle)
-    log.info("Result: soc/range=" + str(soc)+"/" + str(range) + "@" + soc_ts)
+    soc, range, soc_ts, kms, vehicle12v = api.fetch_soc(config, vehicle)
```

Comment:
Darauf wird nicht mehr zugegriffen.
```suggestion
    soc, range, soc_ts = api.fetch_soc(config, vehicle)
```